CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 1.6.1 - 2019-11-25
```

Comment:
this is a new feature and should lead to 1.7.0

please replace the date with "unreleased"

Reply:
done; also replaced 1.6.1 with 1.7.0; not quite sure, what is right here. feel free to enlighten me :-)

Reply:
that looks good, thanks.